_connect-files/api/objects/form-properties/sources/saas/darksky-object.md

```diff
@@ -0,0 +1,71 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/connect-templates/destination-form-property/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+
+# -------------------------- #
+#        CONTENT TYPE        #
+# -------------------------- #
+
+content-type: "api-form"
+form-type: "source"
+key: "source-form-properties-darksky-object"
+
+
+# -------------------------- #
+#        OBJECT INFO         #
+# -------------------------- #
+
+title: "Darksky Source Form Property"
+api-type: "platform.darksky"
+display-name: "Darksky"
+
+source-type: "saas"
+docs-name: "darksky"
+
+property-description: ""
+## Used to create a description for the object that doesn't adhere to the standard in _developers/connect/api/documentation/api-form-properties.html
+## See the Heap object for an example
+
+
+# -------------------------- #
+#      OBJECT ATTRIBUTES     #
+# -------------------------- #
+
+uses-start-date: true
+
+# Only source-specific attributes need to be listed here.
+# The following attributes are considered common,
+# and therefore don't need to be listed:
+# anchor_time, cron_expression, frequency_in_minutes, image_version, start_date 
+
+object-attributes:
+  - name: "language"
+    type: "string"
+    required: true
+    description: "The language."
+    value: "en"
+
+  - name: "location_list"
+    type: "string"
+    required: true
+    description: "The longitude and latitude of the location the app was used."
```

Comment:
Like the UI, I'm assuming that a user could provide more than one location for this property. Is that the case, @jazsmi? If so, this description should be adjusted to reflect that and include the format that values have to be in to be accepted. Ex: `<latitude>,<longitude>` for a single location, and `<latitude>,<longitude>;<latitude>,<longitude>; ... etc` for multiple.